src/util/strings.go

```diff
@@ -0,0 +1,13 @@
+package util
```

Comment:
no `util`/`helper` packages
these should be put somewhere more descriptive

Reply:
Where would you put generic algorithms instead? `kubeobjects`?

Reply:
If this is needed, we'd need to find a name for a generic helper package similar to `kubeobjects`

Reply:
Removed package util and the `Tokenize()` function (replaced with bare `FieldsFunc()`).